### PR TITLE
METAL-800: Retry on failure to reach OFCIR

### PR DESCRIPTION
**Pull request by derekhiggins**

### CODE REVIEW

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

File: ci-operator/step-registry/ofcir/acquire/ofcir-acquire-commands.sh [hunk 1]
```diff
@@ -76,15 +76,17 @@ function getCIR(){
     IPFILE=$SHARED_DIR/server-ip
     CIRFILE=$SHARED_DIR/cir
 
-    if ! timeout 10s curl -kfX POST -H "X-OFCIRTOKEN: $OFCIRTOKEN" "$OFCIRURL?name=$JOB_NAME/$BUILD_ID&type=$CIRTYPE" -o $CIRFILE ; then
```
Comment (dtantsur) [inline ci-operator/step-registry/ofcir/acquire/ofcir-acquire-commands.sh:79]: Did you intend to remove timeout here? Maybe set it to 120  to at least make sure the job does not hang forever?

Comment (derekhiggins) [reply ci-operator/step-registry/ofcir/acquire/ofcir-acquire-commands.sh:79]: Yes, I did intend to remove it, it seemed pointless if retrying after 60 seconds, but your correct, I'll put it back in with an increased timeout

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

**Pull request merged**